test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -122,8 +121,11 @@ static void test(Path top) throws IOException {
             Instant plusHour = Instant.now().plusSeconds(60L * 60L);
             Files.setLastModifiedTime(file, FileTime.from(plusHour));
             FileTime current = creationTime(file);
-            if (!current.equals(creationTime))
+            if (!current.equals(creationTime)) {
+                System.out.println("current = " + current);
```

Comment:
I think it would be better to use `System.err` on these two lines, as was done at line 81.

Reply:
Thanks for the review. These two `System.out` has been replaced as `System.err`.